docs/hello_nf-core/04_make_module.md

```diff
@@ -490,78 +568,147 @@ However, you might want to keep it as a reference for understanding the differen
 
 ---
 
-## 2. Creating modules with nf-core tooling
+## 2. Use nf-core tooling to create modules
 
-In this tutorial, we manually adapted the `cowpy` module step-by-step to teach the nf-core conventions. However, **in practice, you'd use the nf-core tooling to generate properly structured modules from the start**.
+Now that you understand the nf-core module patterns by applying them manually, let's look at how you'd create modules in practice.
```

Comment:
"Now that you understand the nf-core module patterns by applying them manually" feels wrong

Maybe "Now that you've learnt"?